Fix "implicit declaration of function 'openpty' is invalid in C99"
Reverts https://github.com/toddr/IO-Tty/commit/b59e8e2b6f001a135d49cb8f10a9ca4d49a1d5c5
See https://github.com/toddr/IO-Tty/issues/25
--- Tty.xs.orig	2020-01-18 22:49:56.000000000 -0600
+++ Tty.xs	2020-09-03 12:52:37.000000000 -0500
@@ -58,9 +58,7 @@
 #endif /* HAVE_UTIL_H */
 
 #ifdef HAVE_UTIL_H
-# if ((PATCHLEVEL < 19) && (SUBVERSION < 4))
 #  include <util.h>
-# endif
 #endif /* HAVE_UTIL_H */
 
 #ifdef HAVE_PTY_H
